src/chain.h

```diff
@@ -174,7 +174,7 @@ class CBlockIndex
     //! to the genesis block or an assumeutxo snapshot block have reached the
     //! VALID_TRANSACTIONS level.
     //! Change to 64-bit type before 2024 (assuming worst case of 60 byte transactions).
```

Comment:
64d35be20c43f764e5c22657085a790d0adcfe22 should remove this todo comment

Reply:
Done